src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/AspNetStartup.cs

```diff
@@ -20,6 +24,10 @@ public AspNetStartup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            var connectionStrings = new ConnectionStrings();
+            var redis = ConnectionMultiplexer
```

Comment:
Unless you have redis running at the configured location, you'll want to bypass this locally.